lib/util.js

```diff
@@ -33,7 +33,6 @@ const { TextDecoder, TextEncoder } = require('internal/encoding');
 const { isBuffer } = require('buffer').Buffer;
 
 const types = internalBinding('types');
-Object.assign(types, require('internal/util/types'));
```

Comment:
Can we remove the other one instead? This is where `types` is exported, so it’s closer to where people might look for its implementation?